ui/src/main/js/model/job-history-listener.js

```diff
@@ -28,7 +28,7 @@ exports.listen = function (jobUrl, callback) {
     restApi.getJobRuns(jobUrl, function (jobRunsData) {
         callback(jobRunsData);
         setupJobPoll(jobUrl, callback, jobRunsData)
-    });
+    }, {fullStages: 'true'});
```

Comment:
Had to go digging in stapler + the beanutils converter (ugh) but confirmed that's true.